Fails to build in RelWithDebInfo

```
/usr/bin/ld: CMakeFiles/orientation_filter_tests.dir/src/orientation_tools.cpp.o: in function `tf2::impl::toQuaternion(geometry_msgs::msg::Quaternion_<std::allocator<void> > const&)':
/opt/ros/humble/include/tf2/tf2/impl/utils.h:52: undefined reference to `tf2::fromMsg(geometry_msgs::msg::Quaternion_<std::allocator<void> > const&, tf2::Quaternion&)'
/usr/bin/ld: CMakeFiles/orientation_filter_tests.dir/test/orientation_tools_tests.cpp.o: in function `tf2::impl::toQuaternion(geometry_msgs::msg::Quaternion_<std::allocator<void> > const&)':
/opt/ros/humble/include/tf2/tf2/impl/utils.h:52: undefined reference to `tf2::fromMsg(geometry_msgs::msg::Quaternion_<std::allocator<void> > const&, tf2::Quaternion&)'
```

--- a/graceful_controller_ros/src/orientation_tools.cpp
+++ b/graceful_controller_ros/src/orientation_tools.cpp
@@ -38,6 +38,7 @@
 
 #include <angles/angles.h>
 #include <rclcpp/logging.hpp>
+#include <tf2_geometry_msgs/tf2_geometry_msgs.hpp>
 #include <tf2/utils.h>  // getYaw
 #include "graceful_controller_ros/orientation_tools.hpp"
 

--- a/graceful_controller_ros/test/orientation_tools_tests.cpp
+++ b/graceful_controller_ros/test/orientation_tools_tests.cpp
@@ -37,6 +37,7 @@
  *********************************************************************/
 
 #include <gtest/gtest.h>
+#include <tf2_geometry_msgs/tf2_geometry_msgs.hpp>
 #include <tf2/utils.h>  // getYaw
 #include "graceful_controller_ros/orientation_tools.hpp"
 

